payments/voucher_manager.go

```diff
@@ -128,3 +128,22 @@ func (vm *VoucherManager) Balance(channelId types.Destination) (Balance, error)
 	return data.currentBalance, nil
 
 }
+
+// Voucher returns the latest sent voucher for a channel
+func (vm *VoucherManager) Voucher(channelId types.Destination, pk []byte) (Voucher, error) {
```

Comment:
This seems like a getter function but it's actually signing a voucher for you. Probably should clarify that.

Reply:
This is cleaned up in [f501b71](https://github.com/statechannels/go-nitro/pull/826/commits/f501b71e0f665ab4fb58165ccfc253679f5dc248)